docs/source/share_kedro_viz.md

```diff
@@ -109,3 +109,17 @@ You can control who can view your visualisation using [bucket and user policies]
 You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
 
 See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 
+
+## Platform-agnostic sharing with Kedro-Viz 
+
+In Kedro-Viz version 7.1.0, we introduced the `kedro viz build` command that enables you to publish and share Kedro-Viz to any static website hosting platform. Running this command from the command line interface (CLI) creates a `build` folder within the Kedro project. The build folder contains a static Kedro-Viz app package, which can be used as a source to publish on any static website hosting platform.
+
+### Static website hosting platforms such as GitHub Pages
+
+Follow the steps [listed in the GitHub pages documentation](https://docs.github.com/en/pages/quickstart) to create a Git repository that supports GitHub Pages. On completion, push the contents of the `build` folder to this new repository. Your site will be available at the following URL: `http://<username>.github.io`
+
+### Cloud providers such as Azure, and GCP
```

Comment:
[Nit] Can we include AWS as well and then at the end say For AWS, we also offer automated deployment ?

Reply:
For instance, we recently had a situation where someone was deploying to AWS. However, they encountered an issue because they had CloudFront enabled and were using their own DNS. As a result, they were unable to use the direct deploy method.

It's too early to say but I think direct deploy might cause issues. `kedro build` way might be the more robust chosen approach.